### DISCUSSION TITLE
Version 1.0

### DISCUSSION THREAD

**Discussion by JonasSchaub** [Steinbeck-Lab/ChemIcal-DatasEt-comparatoR#17] [General]
Here, we can discuss and compile the things we still want to get done before a first "real" release of CIDER.

**Comment by JonasSchaub** [comment]
Add explicit RDKit version in conda installation (2020.09.1 is currently used but better replace it with most recent one) -> after testing everything in unit tests with the most recent version

**Reply by JonasSchaub** [reply]
This goes for basically every library!

**Reply by JonasSchaub** [reply]
Maybe dockerize it then...

**Comment by JonasSchaub** [comment]
Turning the workflow notebook into documentation on “read the docs”

**Reply by JonasSchaub** [reply]
@Kohulan

**Reply by JonasSchaub** [reply]
Done

**Comment by JonasSchaub** [comment]
- SMILES file import
- Make the PDF export nice
- Maybe also parallelization for some tasks like the chemical space visualisation

**Reply by hannbus** [reply]
The SMILES file import is working now.

**Reply by JonasSchaub** [reply]
That's great!

**Comment by JonasSchaub** [comment]
If parallelization is not applicable easily, push it back to v1.1
